src/pymovements/gaze/integration.py

```diff
@@ -69,7 +85,59 @@ def from_numpy(
     -------
     py:class:`~pymovements.GazeDataFrame`
     """
-    df = pl.from_numpy(data=data, schema=schema, orient=orient)
+    if data is not None:
+        checks.check_is_mutual_exclusive(data=data, time=time)
```

Comment:
add a comment what is tested in the lines...

Reply:
Thanks, good idea!

I will also add an example for each case to the docstring.

Reply:
done